app/controllers/agent/ManageAgentController.scala

```diff
@@ -178,7 +179,7 @@ class ManageAgentController @Inject()(
       )
   }
 
-  private def joinOldJourney(agentCode: Long) =
+  private def joinOldAgentAppointJourney(agentCode: Long) =
     Redirect(
       controllers.agentAppointment.routes.AppointAgentController.getMyOrganisationPropertyLinksWithAgentFiltering(
         pagination = PaginationParameters(page = 1, pageSize = 15),
```

Comment:
PaginationParameters() defaults to PaginationParameters(page = 1, pageSize = 15)